packages/python/javdb_platform/db.py

```
@@ -1818,21 +2704,441 @@ def db_create_report_session(
+        'MovieHistoryAudit': 0,
+        'TorrentHistoryAudit': 0,
+        'MovieHistory.deleted': 0,
+        'MovieHistory.restored': 0,
+        'TorrentHistory.deleted': 0,
+        'TorrentHistory.restored': 0,
+        'MovieHistory.reinserted': 0,
+        'TorrentHistory.reinserted': 0,
+        'drift_skipped': 0,
+    }
+    with get_db(db_path or HISTORY_DB_PATH) as conn:
+        for kind, audit_table, main_table in (
```

Ruff (0.15.12)

[warning] Loop control variable `kind` not used within loop body

Rename unused `kind` to `_kind`

(B007)